org.eclipse.jdt.launching.javaagent/src/main/java/org/eclipse/jdt/launching/internal/weaving/ClassfileTransformer.java

```diff
@@ -14,18 +14,18 @@
  *******************************************************************************/
 package org.eclipse.jdt.launching.internal.weaving;
 
-import org.objectweb.asm.ClassReader;
-import org.objectweb.asm.ClassVisitor;
-import org.objectweb.asm.ClassWriter;
-import org.objectweb.asm.Opcodes;
+import org.eclipse.jdt.launching.internal.org.objectweb.asm.ClassReader;
```

Comment:
It was throwing some errors with existing imports, so did quick fix for that. should I revert that then ?

Reply:
i wonder why it compiles on CI but not for me locally:
![image](https://github.com/user-attachments/assets/f2979041-6854-44ac-8308-d9114092a499)

Reply:
If possible could you try the same ? generating a new javaagent-shaded file, maybe I did some mistake while doing it..

Reply:
probably... that jar is wrong:
![image](https://github.com/user-attachments/assets/e9d92dd8-e9bb-4dc0-8096-00861aa2a39e)

Reply:
when i create the jar it looks the same. Also Readme.md describes that
`Note that ASM classes are _relocated_ to `org.eclipse.jdt.launching.internal.org.objectweb.asm` package to avoid possible conflicts with applicates being debugged.`
however it should not change the source and i still don't get why the CI does not complain about it.

Reply:
i guess org.eclipse.jdt.launching.javaagent is missing in pom.xml so that it is not compiled by CI

Reply:
> when i create the jar it looks the same. Also Readme.md describes that `Note that ASM classes are _relocated_ to `org.eclipse.jdt.launching.internal.org.objectweb.asm` package to avoid possible conflicts with applicates being debugged.` however it should not change the source and i still don't get why the CI does not complain about it.


so is it some issue with the jar ?

Reply:
I have changed the imports to defaults now